axelrod/tests/strategies/test_prober.py

```diff
@@ -232,24 +252,33 @@ class TestNaiveProber(TestPlayer):
     }
 
     def test_strategy(self):
-        # Randomly defects and always retaliates like tit for tat.
         self.first_play_test(C)
         # Always retaliate a defection
-        self.responses_test([D], [C] * 2, [C, D])
+        actions = [(C, D)] + [(D, D)] * 10
```

Comment:
Is this one equivalent? The original test had a cooperation then a defection by the opponent. If we want to replace the test then one with alternator seems more appropriate.